Zend/zend_vm_def.h

```diff
@@ -9817,16 +9817,33 @@ ZEND_VM_HANDLER(210, ZEND_FETCH_DEFAULT_ARG, UNUSED|NUM, UNUSED)
 	ZVAL_UNDEF(EX_VAR(opline->result.var));
 
 	zend_function *called_func = EX(call)->func;
-
 	reflection_parameter_reference param;
-	param.offset = opline->op1.num - 1;
+	param.offset = (uint32_t) -1;
+
+	zend_string *arg_name;
+	if (OP1_TYPE == IS_CONST) {
+		arg_name = Z_STR_P(RT_CONSTANT(opline, opline->op1));
```

Comment:
You could probably use `zend_get_arg_offset_by_name()` here instead of looking for the index yourself, which contains caching of the index. You'll need to allocate a cache slot for this though.

Reply:
I tried this, but it gives me _SSA integrity verification failed_ allocating the cache slot this way:

```c
opline->result.num = zend_alloc_cache_slot();
```

Reply:
Goddamit, I even knew that as well, _and_ I even checked to see if `znode_op` contained a union. I saw this:

```c
	uint32_t      constant;
	uint32_t      var;
	uint32_t      num;
```

And so I assumed it was fine because they're not wrapped in `union {}` (I thought I was looking at a struct declaration). I only just realised now that the entire definition is a union and not a struct (never seen that before; only seen unions embedded in structs).